src/clients/graphql/__tests__/graphql_client.test.ts

```diff
@@ -203,6 +203,28 @@ describe('GraphQL client', () => {
       data: JSON.stringify(query),
     }).toMatchMadeHttpRequest();
   });
+
+  it('properly types the response', async () => {
```

Comment:
Does this test make sense - i.e. is it actually going to test anything sane at run time?

Reply:
Unfortunately I don't think so 😅 but if tsc is set to include test files it will type check the file in the compile step.

Reply:
We have usage of these types here and there so the type sanity will be checked eventually. I'll remove the pointless test :)